dev-tools/hs-backend-booster-dev-tools.cabal

```diff
@@ -0,0 +1,184 @@
+cabal-version: 1.12
```

Comment:
the cabal file should not be checked in

Reply:
if not, we should remove

```
-- This file has been generated from package.yaml by hpack version 0.35.2.
--
-- see: https://github.com/sol/hpack
```

Reply:
Good catch, thanks! Deleting this file (and adding it to `.gitignore`)

Reply:
Well well... that makes the nix build for the integration tests fail, though. Looking into it...